src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/ServiceCategory.kt

```diff
@@ -14,4 +17,9 @@ data class ServiceCategory(
   @CreationTimestamp val created: OffsetDateTime,
   val name: String,
   @OneToMany @JoinColumn(name = "service_category_id") val complexityLevels: List<ComplexityLevel>,
+
+  @ElementCollection()
+  @CollectionTable(name = "service_category_outcome", joinColumns = [JoinColumn(name = "service_category_id")])
+  @Column(name = "outcome")
+  var outcomes: List<String>
```

Comment:
this should be a defined entity i.e. `List<Outcome>`. this follows the pattern used above for complexity levels. it also allows us to easily access the outcome id in this object, instead of just the outcome description. this is required in the DTO to satisfy the front end pacts for service category too.

Reply:
Updated